ethereum/blockcreation/src/main/java/org/hyperledger/besu/ethereum/blockcreation/AbstractBlockCreator.java

```diff
@@ -132,12 +135,23 @@ public Block createBlock(
     return createBlock(Optional.of(transactions), Optional.of(ommers), timestamp);
   }
 
-  private Block createBlock(
+  @Override
+  public Block createBlock(
       final Optional<List<Transaction>> maybeTransactions,
       final Optional<List<BlockHeader>> maybeOmmers,
       final long timestamp) {
+    return createBlock(maybeTransactions, maybeOmmers, Optional.empty(), timestamp, true);
+  }
+
+  protected Block createBlock(
+      final Optional<List<Transaction>> maybeTransactions,
+      final Optional<List<BlockHeader>> maybeOmmers,
+      final Optional<Bytes32> maybeRandom,
+      final long timestamp,
+      boolean rewardCoinbase) {
```

Comment:
actually Gary is the right person since he wrote the code on the merge branch, I am helping porting it on main, in any case I can answer this as well, `rewardCoinbase` [could be false in the `merge` branch](https://github.com/hyperledger/besu/blob/merge/consensus/merge/src/main/java/org/hyperledger/besu/consensus/merge/blockcreation/MergeBlockCreator.java#L67), that part is not yet ported on main, but will be soon